src/screens/HomeScreen.js

```diff
@@ -146,104 +231,65 @@ export const HomeScreen = ({ navigation, route }) => {
     }
   };
 
-  return (
-    <SafeAreaViewFlex>
-      <ScrollView
-        refreshControl={
-          <RefreshControl
-            refreshing={refreshing}
-            onRefresh={refresh}
-            colors={[colors.accent]}
-            tintColor={colors.accent}
-          />
-        }
-      >
-        <ConnectedImagesCarousel
-          alternateAspectRatio
-          navigation={navigation}
-          publicJsonFile="homeCarousel"
-          refreshTimeKey="publicJsonFile-homeCarousel"
+  if (isShow && !!categoriesNews) {
+    return categoriesNews.map(
+      (
+        {
+          categoryButton,
+          categoryId,
+          categoryTitle,
+          categoryTitleDetail,
+          indexCategoryIds,
+          rootRouteName
+        },
+        index
+      ) => (
+        <HomeSection
+          key={index}
+          {...{
+            buttonTitle: categoryButton,
+            categoryId: categoryId,
+            fetchPolicy,
+            navigation,
+            navigate: () =>
```

Comment:
why have you changed the way of passing the props? and how come, that you mix the style of `key` and the others?

Reply:
Because some of the props here come from the map function. So I had to change it.

The reason I wrote key outside the object was to remove the error `Missing "key" prop for element in iterator`.